Update Haskell package set (plus other fixes)

peti opened the pull request:
This PR is test-built by Hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates. I'll fix up the remaining errors and merge it on Friday, 2020-01-17 [20:00 CET](http://www.thetimezoneconverter.com/?t=20:00&tz=Berlin). You can watch this live on Twitch: https://www.twitch.tv/peti343. If you'd like to chat with me (and others) during the stream, you're welcome to use the Twitch chat or join the [`#haskell4nix` voice conference](https://discord.gg/YTEa3XR).

peti committed:
hackage-packages.nix: automatic Haskell package set update

This update was generated by hackage2nix v2.15.0-14-gb942b6a from Hackage revision
https://github.com/commercialhaskell/all-cabal-hashes/commit/b2812edaaed9f9033e48dfbcd45e7e5fb56daccd.

cdepillabout committed:
spago: 0.13.0 -> 0.13.1

peti committed:
hackage2nix: honor LTS for lambdabot, but not for cabal2spec

cdepillabout commented:
@peti We might get LTS-15 with ghc-8.8 sometime in the next two or three weeks.  I guess there might be a lot of breakage in our haskell package set, so if there is anything you'd like me to help with, please feel free to ping me on here.

peti committed:
all-cabal-hashes: update to Hackage at 2020-01-17T18:23:06Z

peti committed:
haskell-doctemplates: provide the latest version when building pandoc 2.9.x

peti committed:
haskell-ghc-lib-parser: update to latest version for hlint

peti committed:
haskell-tls-session-manager: fix build with ghc-8.8.x

peti committed:
hackage-packages.nix: automatic Haskell package set update

This update was generated by hackage2nix v2.15.0-14-gb942b6a from Hackage revision
https://github.com/commercialhaskell/all-cabal-hashes/commit/d1d0ee229831bf9928e884344821c9a27aa4d3d5.

peti committed:
haskell-dhall: update overrides for the new release

I dropped the override for dhall-to-cabal, because the package
is marked as broken already.

peti committed:
hackage2nix: update list of broken packages

peti committed:
haskell-ormolu: fix override for ghc-lib-parser

peti commented:
Next week we'll merge https://github.com/NixOS/nixpkgs/pull/78380.
